packages/plugin-conversation/test/integration/spec/verbs.js

```diff
@@ -94,8 +95,45 @@ describe(`plugin-conversation`, function() {
         }));
     });
 
+    describe(`#assign()`, () => {
+      before(() => spark.conversation.create({participants})
+        .then((c) => {
+          conversation = c;
+        }));
+
+      let sampleImageSmallOnePng = `sample-image-small-one.png`;
+
+      before(() => fh.fetch(sampleImageSmallOnePng)
+        .then((res) => {
+          sampleImageSmallOnePng = res;
+        }));
+
+      it(`assigns an avatar to a room`, () => spark.conversation.assign(conversation, sampleImageSmallOnePng)
+        .then(() => spark.conversation.get(conversation))
+        .then((c) => {
+          assert.property(c, `avatar`);
+
+          assert.property(c.avatar, `files`);
+          assert.property(c.avatar.files, `items`);
+          assert.lengthOf(c.avatar.files.items, 1);
+          assert.property(c.avatar.files.items[0], `fileSize`);
+          assert.property(c.avatar.files.items[0], `mimeType`);
+          assert.property(c.avatar.files.items[0], `objectType`);
+          assert.property(c.avatar.files.items[0], `scr`);
+          assert.property(c.avatar.files.items[0], `url`);
+          assert.equal(c.avatar.objectType, `content`);
```

Comment:
I would say we could make this a custom asserter, but I don't anticipate us doing this much.